maestro-driver-ios/maestro-driver-iosUITests/Routes/XCTest/PointerEventPath.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+struct PointerEventPath {
```

Comment:
nit : better feels like class, because `let path: NSObject` reference type is used inside

Reply:
I don't agree here: The `XCPointerEventPath` would nowadays have been implemented as a struct. There is no downside of wrapping a reference type in a struct. The struct api is more safe and convenient.

Reply:
yeah, it is just a nit, I used to prefer clean value types back then when coding in Swift (may be my knowledge is rusty :D), but there is no bad side-effects of using reference type inside, except that copying + mutating it might bring some unexpected mutations for the reference.

Reply:
Totally agree on the surprising mutating behaviour. Updated the code to use `mutating func` where the struct is mutating the underlying value type.